app/controllers/home_controller.rb

```diff
@@ -1,3 +1,9 @@
 class WeekFourAssessmentApp
-  # Your code will go here.
+  get '/' do
+    @name = 'Sophie'
+    erb :welcome
+  end
+  get '/name/:name' do
+    user = User.create(name: params[:name])
```

Comment:
I don't know if I missed this in my first review, but you need to render some html (erb)